service/implementation/email_sender/user_subscription_manager.py

```diff
@@ -0,0 +1,99 @@
+import os
+import base64
+
+from dependency_injector.wiring import Provide, inject
+from google.oauth2.credentials import Credentials
+from google_auth_oauthlib.flow import InstalledAppFlow
+from google.auth.transport.requests import Request
+from googleapiclient.discovery import build
+from email.mime.text import MIMEText
+
+from database.models import TempSubscribersData
+
+from pathlib import Path
+from dotenv import load_dotenv
+
+from sqlalchemy import event
+
+from api.container.container import Container
+from database.postgres.dal.user_subscription import UserSubscriptionDAL
+
+
+class UserSubscriptionManagerMeta(type):
```

Comment:
I already saw pretty much the same chunk of code before. Do we need that duplication?